### ISSUE TITLE
Developing space branding

### ISSUE THREAD

**Issue by sirinoks** [DevelopingSpace/.github#1]
*We haven't yet confirmed if this will be the place for universal decisions regarding DS, if we change it - I will move this issue*

This is a large issue. It involves creating a specific style to all DS projects, and rules for the future ones.

Rules for logos, colors, themes. 

I want people to participate, toss out their ideas, and discuss. 

This is what I got so far, I will add stuff here as I come up with more.

## Discussions ##
- ### Using letters vs tools ###
We have a Telescope logo as an example of having both a letter and a space tool. However, we could just say - either use a tool, or a letter. Perhaps, have different requirements of both. It could be difficult to combine the two in future projects.

- ### Is Sattelite big enough to have its own logo? ###

Reasons to touch Satellite:
1. I have a simple idea for a logo, which would work with the current branding guidelines ideas. 
2. It is a separate repo,

Reasons to not touch Satellite: 
1. No user view, so no reason to have a logo.
2. It uses S like Starchart, and we have an abundance of S-es already.

- ### We have a problem. There are 3 projects already that use a letter S. Starchart, Satellite, developing Space. I might just focus on D for DS, but S has so much cooler stuff.... ###

# Branding #

### Color ###
We need a single color for the background that is consistent, which will be the responsible for the Developing Space

Each project will additionally have their own accent color and a complementary color for the background. Accent color will show up throughout the app, might be the primary color of the logo (foreground object). 
Complementary color will fill up half of the triangle formed by the diagonal line of the color. 

## Logo style guide ##
Letter, space related item (technology?), 2 diagonal lines.

### Diagonal lines ###
Keep in mind that the two lines need to be opposite, and can be interchanged.
- Should we have some meaning in having lines a specific way? For example, in all projects that have a "face" to show the users (a website view), will have it like `/` as the main object (Like the current Telescope logo). While all the additional tools (Satellite), back end solutions, or organisational elements (Developing Space) will have the main object this way `\`

1st diagonal needs to be used in the formation of the foreground object
<img src="https://user-images.githubusercontent.com/13803161/218919063-025f3f16-f990-48e1-abeb-7dd7694c59e3.png" width="50%" /> 

Example:
A foreground item that is aligned with the diagonal line
<img src="https://user-images.githubusercontent.com/13803161/218919855-6035e742-d5a2-4ea4-b7a4-3d17e623f65c.png" width="50%" />

2nd diagonal is for the background color variation
<img src="https://user-images.githubusercontent.com/13803161/218919277-5b9b76c6-8690-43bb-b2f2-22e64e91c572.png" width="50%" /> 

Example:
1 half of the triangle formed is filled with a background color of the app
<img src="https://user-images.githubusercontent.com/13803161/218920470-33bd3e8d-0655-4a44-89c0-7ff089e58c0b.png" width="50%" />

## Logo ideas ##
2 black holes colliding, S

- For DS? 
 
<img src="https://user-images.githubusercontent.com/13803161/218918365-1f8184ce-88d8-4c96-9834-efc82ee5f480.png" width="50%" />
<img src="https://user-images.githubusercontent.com/13803161/218918407-2d9bb2b5-9711-41b5-ac3d-1e4db105ef3c.png" width="50%" />
<img src="https://user-images.githubusercontent.com/13803161/218918350-74443e4f-780b-4e84-8292-c46356c675fd.png" width="50%" />

### Satellite ###
Logo idea (ignore colors):

<img src="https://user-images.githubusercontent.com/13803161/218924969-deed0216-a1df-4ea9-aee5-70e8c379a30f.png" width="50%" />

*Basically, this:*
<img src="https://user-images.githubusercontent.com/13803161/218926294-498ebb9f-e9cd-4f7c-85c7-97761f5f49d1.png" width="50%" />

**Comment by Genne23v** [comment]
How about making one universal logo and adding each project title on the bottom of the logo? That way, branding can be more powerful as we have the consistent logo across project.
